sdk/eventhub/event-hubs/src/eventHubConsumerClient.ts

```diff
@@ -35,14 +35,21 @@ const defaultConsumerClientOptions: Required<Pick<
 };
 
 /**
- * The `EventHubConsumerClient` is the main point of interaction for consuming events in Azure Event Hubs service.
+ * The `EventHubConsumerClient` is the main point of interaction for consuming events from an
+ * Event Hub instance.
  *
  * There are multiple ways to create an `EventHubConsumerClient`
  * - Use the connection string from the SAS policy created for your Event Hub instance.
  * - Use the connection string from the SAS policy created for your Event Hub namespace,
  * and the name of the Event Hub instance
- * - Use the fully qualified domain name of your Event Hub namespace like `<yournamespace>.servicebus.windows.net`,
+ * - Use the fully qualified version of the namespace like `<yournamespace>.servicebus.windows.net`,
```

Comment:
I don't like this change. FQDNs are well-known things. Fully qualified namespaces might mean anything.

Reply:
I understand that fully-qualified namespace is an Event Hubs term so it's probably fine. I'm still a little sad about it.

Reply:
How about 

> Use the full namespace like `<yournamespace>.servicebus.windows.net`

Reply:
I like that.